eng/common/TestResources/New-TestResources.ps1

```diff
@@ -65,7 +65,8 @@ param (
     [hashtable] $AdditionalParameters,
 
     [Parameter()]
-    [hashtable] $EnvironmentVariables,
+    [ValidateNotNullOrEmpty()]
```

Comment:
This works (won't require the parameter, just that it contains data), but you might want to allow empty. Pointless to pass it, but more permissions:
```suggestion
    [ValidateNotNull()]
```

Reply:
Based on how powershell works, if we pass it in as empty, will it then not get initialized by the default value?

Reply:
It still has to match the parameter type (if declared, like it is), so empty would be `@{}` which matches the default.